Various/rodilab_Color palette.lua

```diff
@@ -548,6 +543,10 @@ function set_window()
   ctx = r.ImGui_CreateContext(script_name, r.ImGui_ConfigFlags_DockingEnable())
   font = r.ImGui_CreateFont('Arial',12)
   r.ImGui_AttachFont(ctx,font)
+  if conf.mouse_pos then
+    cur_x, cur_y = r.GetMousePosition()
+    cur_x, cur_y = r.ImGui_PointConvertNative(ctx, cur_x, cur_y, true)
```

Comment:
`to_native` must be false (or omitted, false is the default) when converting from platform-native coordinates to ImGui coordinates. (Otherwise HiDPI scaling will be applied in reverse.)